data/raw/new_orleans_csc/appeal_hearing_pdfs/nopd_appeals_spacy_model.12_25_25.model/meta.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lang":"en",
+  "name":"model",
+  "version":"0.0.0",
+  "spacy_version":">=2.3.7",
+  "description":"",
+  "author":"",
+  "email":"",
+  "url":"",
+  "license":"",
+  "spacy_git_version":"cae72e46d",
+  "vectors":{
+    "width":0,
+    "vectors":0,
+    "keys":0,
+    "name":"spacy_pretrained_vectors"
+  },
+  "pipeline":[
+    "ner"
+  ],
+  "factories":{
+    "ner":"ner"
+  },
+  "labels":{
+    "ner":[
+      "accused_name",
+      "allegation",
+      "appeal_date",
+      "appeal_disposition",
+      "attorney_name",
+      "docket_number",
+      "incident_date",
+      "incident_summary",
+      "initial_action",
+      "nopd_incident_report_number",
+      "notification_of_action_date"
+    ]
+  }
+}
```

Comment:
Rather keeping the model in Git, I think it's better to package the model using `python -m spacy package` and save it on Dropbox if you want to keep it. Or just git ignore it all if you're still improving the model.

Reply:
In the new framework, models will be regarded as a kind of data, not a kind of code. Meaning it will not be kept in Git but in a data repository of some kind (Dropbox?) to be pulled like how you're pulling raw CSVs right now, or generated from scratch.

Reply:
All of my work on improving the model is local, so there is no need for this branch to exist for the moment. My intention with this branch was to learn from your initial feedback, as I found myself reaching a place in my progress where I'm finally deriving some utility from what I've produced so far. 

I'm going to close this pull request and remove the branch. We can continually revisit its progress as we continue our work on the new pipeline?

Reply:
Sure